<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-1</y>
      <width>881</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Number of rings:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>211</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <column>
     <property name="text">
      <string>Start point</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End point</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="RestartBut">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QLabel" name="stepsLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Steps needed:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
